app/controllers/v1/greetings_controller.rb

```diff
@@ -0,0 +1,15 @@
+class V1::GreetingsController < ApplicationController
+  def index
```

Comment:
- [ ] Kindly work on the function to reflect a project requirement, the greetings should come from the database, in this case, it can be useful to create a function for creating 5 messages(greetings) in a seed file.